### PR TITLE
Add a new P-Ray Scanner, grant to CE, T-Ray is now a Protolathe item

### PR DESCRIPTION
**Pull request by ghost**
- Added a P-Ray scanner. It is basically a T-Ray scanner with boosted range of three tiles instead of 1, the Chief Engineer starts with one, otherwise must be produced via R&D (Engi 3 and EMP 3)
- Moved the T-Ray scanner to the Protolathe, (Engi 1 and EMP 1). Mineral price upped slightly in line with other items
- Removed Autolathe design

Maybe this will compile here, otherwise I'm none-the-wiser as to what isn't working here

### CODE REVIEW

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -8,14 +8,14 @@ REAGENT SCANNER
 */
 
 /obj/item/device/t_scanner
-	name = "T-ray scanner"
+	name = "t-ray scanner"
```
Comment (PJB3005) [inline code/game/objects/items/devices/scanners.dm]: IMO having the T capitalized makes more sense.

Comment (ghost) [inline code/game/objects/items/devices/scanners.dm]: Then I'll mark it as improper I guess

File: code/game/objects/items/devices/scanners.dm [hunk 2]
```diff
@@ -75,6 +75,16 @@ REAGENT SCANNER
 				if(M)
 					M.invisibility = INVISIBILITY_LEVEL_TWO
 
+/obj/item/device/t_scanner/adv
+
```
Comment (PJB3005) [inline code/game/objects/items/devices/scanners.dm]: Whytespace.

Comment (ComicIronic) [inline code/game/objects/items/devices/scanners.dm]: We must secure the existence of our codebase and a future for whitespace

File: code/game/jobs/job/engineering.dm [hunk 3]
```diff
@@ -32,6 +32,7 @@
 			if(4) H.equip_or_collect(new /obj/item/weapon/storage/backpack/satchel(H), slot_back)
 		H.equip_or_collect(new /obj/item/clothing/under/rank/chief_engineer(H), slot_w_uniform)
 		//H.equip_or_collect(new /obj/item/device/pda/heads/ce(H), slot_l_store)
+		H.equip_or_collect(new /obj/item/device/t_scanner/adv(H), slot_r_store)
```
Comment (Kurfursten) [inline code/game/jobs/job/engineering.dm]: Should probably spawn in the CE locker instead, like the advanced device analyzer and advanced tool-belt.

**Comment by Kurfursten** [comment]
It seems weird to put T-ray scanners behind the R&D wall. 

Especially given that they are publicly accessible at the YouTool and that unlike welders I've never seen them sell out in a shift.

**Comment by ComicIronic** [comment]
They're not behind the wall - mag 1 eng 1 is the starting research.

File: code/game/jobs/job/engineering.dm (hunk 3 above)
Comment (ghost) [inline code/game/jobs/job/engineering.dm]: Got it

Comment (ghost) [inline code/game/jobs/job/engineering.dm]: Dun it

**Comment by Kurfursten** [comment]
@ComicIronic I consider the acquisition of a protolathe + R&D console more of a wall than doing research or collecting any amount of metal and glass

But like I said it's just weird it won't actually matter for anything. I can't foresee any future in which I'll need a T-ray scanner and won't have two YouTools full of them.

**Comment by ComicIronic** [comment]
Pic of the sprite please.

**Comment by ComicIronic** [comment]
Also why did you remove the autolathe design? It's not mutually exclusive that a design be on the autolathe and the protolathe.

**Comment by ghost** [comment]
I thought it made sense to make something like that into an R&D item. In the future, it might even do more stuff. For now, the advanced T-Ray (P-Ray) scanner is pretty damn useful for repairs

p-ray

**Comment by Kurfursten** [comment]
It's obsoleted by the optical material scanner right?

**Comment by ghost** [comment]
Possibly, I have not checked

**Comment by PJB3005** [comment]
Technically isn't, but in most cases it is due to how shit the pulsing is. 

The material scanners only show the starting layout (unless you rewrite the holomap), the T-ray shows how the pipes currently are.

**Pull request merged**